<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>settingWLAN</class>
 <widget class="QDialog" name="settingWLAN">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>18</pointsize>
   </font>
  </property>
  <widget class="QLabel" name="labelKey">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>130</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Key</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditKey">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>140</y>
     <width>391</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButtondhcp">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>190</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>DHCP</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButtonStaticIP">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>250</y>
     <width>191</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Static IP</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelIP">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>290</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>IP</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelNetmask">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>390</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Netmask</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelGateway">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>340</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Gateway</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelDNS">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>430</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>DNS server</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditIP">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>280</y>
     <width>381</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditGateway">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>330</y>
     <width>381</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditNetmask">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>380</y>
     <width>381</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditDNS">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>430</y>
     <width>381</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelAP">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>90</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>ESSID</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelEssid">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>90</y>
     <width>481</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonApply">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>220</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Apply</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonCancel">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>130</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonOK">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>0</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="wireless.qrc">
     <normaloff>:/images/back.png</normaloff>:/images/back.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>60</width>
     <height>60</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonKeyBoard">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>400</y>
     <width>91</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="wireless.qrc">
     <normaloff>:/images/keyboard.png</normaloff>:/images/keyboard.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>64</width>
     <height>64</height>
    </size>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="wireless.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButtonCancel</sender>
   <signal>clicked()</signal>
   <receiver>settingWLAN</receiver>
   <slot>cancel()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>390</x>
     <y>445</y>
    </hint>
    <hint type="destinationlabel">
     <x>442</x>
     <y>452</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonOK</sender>
   <signal>clicked()</signal>
   <receiver>settingWLAN</receiver>
   <slot>ok()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>620</x>
     <y>433</y>
    </hint>
    <hint type="destinationlabel">
     <x>626</x>
     <y>404</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEditIP</sender>
   <signal>cursorPositionChanged(int,int)</signal>
   <receiver>settingWLAN</receiver>
   <slot>checkstatic()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>481</x>
     <y>227</y>
    </hint>
    <hint type="destinationlabel">
     <x>571</x>
     <y>234</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEditGateway</sender>
   <signal>cursorPositionChanged(int,int)</signal>
   <receiver>settingWLAN</receiver>
   <slot>checkstatic()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>489</x>
     <y>289</y>
    </hint>
    <hint type="destinationlabel">
     <x>595</x>
     <y>292</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEditNetmask</sender>
   <signal>cursorPositionChanged(int,int)</signal>
   <receiver>settingWLAN</receiver>
   <slot>checkstatic()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>515</x>
     <y>340</y>
    </hint>
    <hint type="destinationlabel">
     <x>570</x>
     <y>337</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEditDNS</sender>
   <signal>cursorPositionChanged(int,int)</signal>
   <receiver>settingWLAN</receiver>
   <slot>checkstatic()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>486</x>
     <y>402</y>
    </hint>
    <hint type="destinationlabel">
     <x>563</x>
     <y>391</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonKeyBoard</sender>
   <signal>clicked()</signal>
   <receiver>settingWLAN</receiver>
   <slot>keyboard()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>301</x>
     <y>433</y>
    </hint>
    <hint type="destinationlabel">
     <x>328</x>
     <y>444</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonApply</sender>
   <signal>clicked()</signal>
   <receiver>settingWLAN</receiver>
   <slot>apply()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>110</x>
     <y>437</y>
    </hint>
    <hint type="destinationlabel">
     <x>177</x>
     <y>424</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>apply()</slot>
  <slot>cancel()</slot>
  <slot>ok()</slot>
  <slot>checkstatic()</slot>
  <slot>keyboard()</slot>
 </slots>
</ui>
